projects/storefrontlib/shared/components/chat-messaging/avatar/avatar.component.html

```diff
@@ -0,0 +1,12 @@
+<!-- TODO: Verify current user to display left aligned messages -->
+<div
+  *ngIf="event"
+  class="cx-avatar"
+  [ngClass]="{ 'right-align': event.addedByAgent }"
+>
+  <h1 *ngIf="!event.addedByAgent">
+    {{ getIntitials(event?.author) }}
```

Comment:
no need to add "?"  
Again, let's give the hosting component to provide the name or the email
```suggestion
    {{ getIntitials(message.author) }}
```